bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/api/ShellyApiInterface.java

```diff
@@ -34,108 +34,112 @@
  */
 @NonNullByDefault
 public interface ShellyApiInterface {
-    boolean isInitialized();
+    public boolean isInitialized();
```

Comment:
Why did you add the `public` keyword in all interfaces? This is redundant and should be left away.

Reply:
you are right, stupid. Removed from all interface declarations
fixed